recipes/eukfinder/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -ex
+
+df -h
+
+# Setup PLAST binaries
+if [[ $(uname) == "Darwin" ]]; then
+  wget https://github.com/PLAST-software/plast-library/releases/download/v2.3.2/plastbinary_osx_v2.3.2.tar.gz
+  tar -zxf plastbinary_osx_v2.3.2.tar.gz
+  cd plastbinary_osx_v2.3.2
+elif [[ $(uname) == "Linux" ]]; then
+  wget https://github.com/PLAST-software/plast-library/releases/download/v2.3.2/plastbinary_linux_v2.3.2.tar.gz
+  tar -zxf plastbinary_linux_v2.3.2.tar.gz
+  cd plastbinary_linux_v2.3.2
+else
+  echo "Unsupported architecture!"
+  exit 1
+fi
+
+cp build/bin/plast $PREFIX/bin/
+cd ..
+
+# Clone acc2tax repository
+git clone https://github.com/richardmleggett/acc2tax.git
```

Comment:
Can you use a tag instead of the default branch ? The default branch is a moving target.